README.md

```diff
@@ -1,3 +1,20 @@
+##Install TL;DR
+
+```shell
+pip install pre-commit detect-secrets
+mkdir .gds
+detect-secrets scan > .gds/.secrets.baseline
+detect-secrets audit .gds/.secrets.baseline
+pre-commit install -c .gds/.pre-commit-config.yaml
```

Comment:
we should add that they'll have to get the .pre-commit-config.yaml from our repo